docs/01-overview/main-areas/observability/obsv-04-telemetry-in-kyma.md

```diff
@@ -0,0 +1,392 @@
+
+## Prerequisites
+
+- The telemetry component is in an alpha state. You must [explicitly install it](./../../../04-operation-guides/operations/obsv-00-enable-telemetry_component.md).
+- Your application must log to `stdout` or `stderr`.
+
+## Architecture
+
+### Telemetry component
+
+The telemetry component provides [Fluent Bit](https://fluentbit.io/) as a log collector. Fluent Bit collects all application logs of the cluster workload and ships them to a backend.
+
+![Telemetry Architecture](./assets/telemetry-logging-arch.drawio.svg)
+
+1. Container logs are stored by Kubernetes container runtime under the `var/log` directory and its subdirectories.
+2. The collector runs as a DaemonSet (one instance per node), detects any new log files in the folder, and tails them using a filesystem buffer for reliability.
+3. The collector queries the [Kubernetes API Server](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/) for additional Pod metadata, such as Pod annotations and labels.
+4. The telemetry component configures Fluent Bit with your custom output configuration.
+5. If Kyma's `logging` component is installed, the operator configures the shipment to the in-cluster Loki instance automatically.
+6. As specified in your custom configuration, Fluent Bit sends the log data to observability systems outside the Kyma cluster.
+7. The user accesses the internal and external observability system to analyze and vizualize the logs.
+
+### Pipelines
+
+Fluent Bit comes with a pipeline concept, which supports flexible combination of inputs with outputs and filtering in between - for details, see [Fluent Bit: Output](https://docs.fluentbit.io/manual/concepts/data-pipeline/output).
+Kyma's telemetry component brings a predefined setup of the Fluent Bit DaemonSet and a base configuration, which assures that the application logs of the workloads in the cluster are processed reliably and efficiently. Additionally, the telemetry component provides a Kubernetes API called `LogPipeline` to configure outputs with some filtering capabilities.
+
+![Pipeline Concept](./assets/telemetry-logging-pipelines.drawio.svg)
+
+1. A central `tail` input plugin reads the application logs.
+
+2. The application logs are enriched by a `kubernetes` filter. Then, for every `LogPipeline` definition, a `rewrite_tag` filter is generated, which uses a dedicated `tag` with name `<logpipeline>.*`, followed by the custom configuration defined in the `LogPipeline` resource. You can add your own filters to the default filters.
```

Comment:
```suggestion
2. The application logs are enriched by a `kubernetes` filter. Then, for every LogPipeline definition, a `rewrite_tag` filter is generated, which uses a dedicated `tag` with name `<logpipeline>.*`, followed by the custom configuration defined in the LogPipeline resource. You can add your own filters to the default filters.
```
Backticks are only needed when referring to code. I think here we refer to the resource name?